### PR TITLE
Date fix

### PR DESCRIPTION
**Pull request by levent91**
Added `minMaxDates` and updated the Input Schema.
Run: https://console.apify.com/actors/dev/FuhVWBiT7w8a9yk7g#/source

### CODE REVIEW

File: INPUT_SCHEMA.json [hunk 1]
```diff
@@ -31,6 +31,30 @@
...
         },
+        "videosFromDate": {
+            "title": "Videos from",
+            "type": "string",
+            "editor": "textfield",
+            "description": "Scrape the videos from the date e.g <br />30 minutes <br />1 week <br />5 months <br />4 years, etc.<br />for 'Earliest' empty the field",
+            "pattern": "(^(1|([^0a-zA-Z ][0-9]{0,3})) (minute|hour|day|week|month|year))s?",
+            "minLength": 0,
+            "maxLength": 16,
+            "prefill": "01-01-2000",
```
Comment (metalwarrior665) [inline INPUT_SCHEMA.json:42]: Where is this format coming in? Standard is to use ISO format, e.g. YYYY-MM-DD so I would use that if we can

**Review by metalwarrior665 — CHANGES_REQUESTED** [review]
@pocesar Can you check if this implementation makes sense?

File: INPUT_SCHEMA.json (hunk 1 above)
Comment (pocesar) [reply INPUT_SCHEMA.json:42]: the pattern should actually be like this https://github.com/gippy/instagram-scraper/blob/726043ae1eaef7af4a524932a1db18b33b35d383/INPUT_SCHEMA.json#L92

omit the `prefill`

File: package.json [hunk 2]
```diff
@@ -14,7 +14,8 @@
         "apify": "^1.2.1",
         "moment": "^2.29.1",
         "node-fetch": "^2.6.1",
-        "puppeteer": "*"
+        "puppeteer": "*",
+        "apify-utils": "https://github.com/metalwarrior665/apify-utils.git#1.0.1"
```
Comment (pocesar) [inline package.json:18]: use `1.1.0`

File: src/main.js [hunk 3]
```diff
@@ -28,6 +30,8 @@ Apify.main(async () => {
     if (verboseLog) {
         log.setLevel(log.LEVELS.DEBUG);
     }
+
+    const minMaxDate = time.minMaxDates({ min: videosFromDate || new Date(Date.now()/1000) , max: videosToDate || new Date()});
```
Comment (pocesar) [inline src/main.js:34]: can leave `undefined`, it won't use it
`minMaxDates({ min: undefined, max: undefined }).compare()` effectively is like a passthrough

**Review by pocesar — CHANGES_REQUESTED** [review]
the current way of picking the date from the list isn't working as expected, so it should be either refactored out or removed entirely. people are having issues with any setting beyond the prefilled one (5 years)

File: package.json (hunk 2 above)
Comment (metalwarrior665) [reply package.json]: We decided to copy the functions before we finalize the package.

**Comment by metalwarrior665** [comment]
@pocesar has vacation so let's push this to beta and @zpelechova can test

**Pull request merged**